website/integrations: add filerise (#14610)
goauthentik/authentik: website/integrations/services/filerise/index.mdx

    - **Application**: provide a descriptive name, an optional group for the type of application, the policy engine mode, and optional UI settings.
    - **Choose a Provider type**: select **OAuth2/OpenID Connect** as the provider type.
    - **Configure the Provider**: provide a name (or accept the auto-provided name), the authorization flow to use for this provider, and the following required configurations.
        - Note the **Client ID**,**Client Secret**, and **slug** values because they will be required later.
        - Set **Redirect URI** to `https://filerise.company/api/auth/auth.php?oidc=callback`.
        - Select any available signing key.
    - **Configure Bindings** _(optional)_: you can create a [binding](/docs/add-secure-apps/flows-stages/bindings/) (policy, group, or user) to manage the listing and access to applications on a user's **My applications** page.

3. Click **Submit** to save the new application and provider.

## FileRise configuration

1. Log in to FileRise as an admin.

> 2. Click on your profile icon at the top right, and then **Admin Panel**.

Reviewer: ```suggestion
2. Click on your profile icon in the upper right corner, then select **Admin Panel**.
```

> 3. Expand the **OIDC Configuration & TOTP** section, and set the following configurations:

Reviewer: ```suggestion
3. Open the **OIDC Configuration & TOTP** section and configure the following settings:
```

:::

## authentik configuration

To support the integration of FileRise with authentik, you need to create an application/provider pair in authentik.

### Create an application and provider in authentik

1. Log in to authentik as an admin, and open the authentik Admin interface.
2. Navigate to **Applications** > **Applications** and click **Create with Provider** to create an application and provider pair. (Alternatively you can first create a provider separately, then create the application and connect it with the provider.)

    - **Application**: provide a descriptive name, an optional group for the type of application, the policy engine mode, and optional UI settings.
    - **Choose a Provider type**: select **OAuth2/OpenID Connect** as the provider type.
    - **Configure the Provider**: provide a name (or accept the auto-provided name), the authorization flow to use for this provider, and the following required configurations.
        - Note the **Client ID**,**Client Secret**, and **slug** values because they will be required later.
        - **Redirect URI**:

>             - Strict: `https://filerise.company/api/auth/auth.php?oidc=callback`

Author: I've changed the format to match other docs. Signing key isn't under advanced settings.

        - **Signing key**: select any available signing key.
    - **Configure Bindings** _(optional)_: you can create a [binding](/docs/add-secure-apps/flows-stages/bindings/) (policy, group, or user) to manage the listing and access to applications on a user's **My applications** page.

3. Click **Submit** to save the new application and provider.

## FileRise configuration

> 1. Log in to FileRise as an admin.

Reviewer: can we be more descriptive? like click the cog wheel or something

Reviewer: meant to add to step 2

2. Click on the **Admin Panel** icon at the top right.
3. Expand the **OIDC Configuration & TOTP** section, and set the following configurations:
    - **OIDC Provider URL**: `https://authentik.company/application/o/<application-slug>/`
    - **OIDC Client OpenID**: Client ID from authentik
    - **OIDC Client Secret**: Client Secret from authentik
    - **OIDC Redirect URI**: `https://filerise.company/api/auth/auth.php?oidc=callback`
4. Click **Save Settings**.

> 

Reviewer: Can we add a resources section?

Author: They don't have a guide for SSO so there's nothing relevant to link to
